exchanges/kraken/kraken_wrapper.go

```diff
@@ -621,7 +622,14 @@ func (k *Kraken) CancelOrder(o *order.Cancel) error {
 		return err
 	}
 	if k.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
-		return k.wsCancelOrders([]string{o.ID})
+		var ordersList []string
```

Comment:
CancelOrder(o *order.Cancel) is primarily assumes singular orderIDs will be passed in. If we start passing in orderIds with comma separated string for some exchanges and not for others, then that becomes an issue. Solution for this would be to add CancelBatchOrders to the wrappers. This would be a big task but to make it easier you could handle kraken properly and for the rest of the exchanges just return nil, common.ErrFunctionNotImplemented

Reply:
done